CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.10)
+
+#project_name
+project(UniversityTimetable)
+
+#Specify the C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ggdb")
+
+#Include directories
+include_directories(include)
+include_directories(argparse)
+
+#Source files for the library
+set(SRC
+    src/Course.cpp
+    src/Genetic.cpp
+    src/Instructor.cpp
+    src/TimeSlot.cpp
+    src/University.cpp
+)
+
+#Create the dynamic library
+add_library(UniversityTimetable SHARED ${SRC})
+
+#Specify where to output the library
+set_target_properties(UniversityTimetable PROPERTIES
```

Comment:
Fix style like set(SRC ...

Reply:
you mean change set_target_properties into set?

Reply:
no, only style for tabs